spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -187,7 +188,7 @@ def _default_file_to_render; end # Stub method
         Class.new do
           include ViewExampleGroup::ExampleMethods
           def controller
-            @controller ||= OpenStruct.new(params: nil)
+            @controller ||= ::OpenStruct.new(params: nil)
```

Comment:
I think this can become just `Struct.new(:params).new(nil)`, and we’re openstruct-free with no side effects.

Reply:
Ok I will submit a PR with that change to start the replacement.

Reply:
@pirj I made this change in #2755